packages/components/src/alert/tests/chromatic/AlertTrigger.chroma.jsx

```diff
@@ -2,30 +2,28 @@ import { Alert, AlertTrigger } from "@components/alert";
 import { Button } from "@components/button";
 import { Content } from "@components/placeholders";
 import { Heading } from "@components/typography";
-import { storiesOfBuilder } from "@stories/utils";
 
-function stories(segment) {
-    return storiesOfBuilder(module, "Chromatic/AlertTrigger")
-        .segment(segment)
-        .build();
-}
+export default {
+    title: "Chromatic/AlertTrigger",
+    component: AlertTrigger
+};
 
-stories()
-    .add("default", () =>
-        <AlertTrigger>
-            <Button variant="secondary">Open</Button>
-            <Alert primaryButtonLabel="Yes">
-                <Heading>Launch</Heading>
-                <Content>Are you sure you want to launch the space shuttle?</Content>
-            </Alert>
-        </AlertTrigger>
-    )
-    .add("default open", () =>
-        <AlertTrigger defaultOpen>
-            <Button variant="secondary">Open</Button>
-            <Alert primaryButtonLabel="Yes">
-                <Heading>Launch</Heading>
-                <Content>Are you sure you want to launch the space shuttle?</Content>
-            </Alert>
-        </AlertTrigger>
-    );
+export const Default = () => <AlertTrigger>
+    <Button variant="secondary">Open</Button>
+    <Alert primaryButtonLabel="Yes">
+        <Heading>Launch</Heading>
+        <Content>Are you sure you want to launch the space shuttle?</Content>
+    </Alert>
+</AlertTrigger>;
+
+
+export const DefaultOpen = () => <AlertTrigger defaultOpen>
+    <Button variant="secondary">Open</Button>
+    <Alert primaryButtonLabel="Yes">
+        <Heading>Launch</Heading>
+        <Content>Are you sure you want to launch the space shuttle?</Content>
+    </Alert>
+</AlertTrigger>;
+
+Default.storyName = "default";
```

Comment:
I believe you should collocate the story additional parameter to it's declaration.

It would be easier for maintenance as there could be many additional parameters per story